cluster/src/main/java/org/apache/iotdb/cluster/server/member/MetaGroupMember.java

```diff
@@ -1326,50 +1326,21 @@ public void receiveSnapshot(SendSnapshotRequest request) throws SnapshotInstalla
    */
   @Override
   public TSStatus executeNonQueryPlan(PhysicalPlan plan) {
```

Comment:
Why not delete this method as you have implemented it in Coordinator.

Reply:
DataGroupMember use the method, so I have to support the method in metaGroupMember.

Reply:
Where does DataGroupMember use this? It does not seem quite reasonable.

Reply:
It seems that we can move  `executeNonQueryPlan` from `RaftService` to `TSDataService` in cluster.thrift

Reply:
ok，I will remove the method.